docs/tutorial/04-non-fungible-tokens.mdx

```diff
@@ -321,33 +347,60 @@ pub contract ExampleNFT {
     }
 
 	init() {
+        // use unique path names
+        self.CollectionStoragePath = /storage/nftTutorialCollection
+        self.CollectionPublicPath = /public/nftTutorialCollection
+        self.MinterStoragePath = /storage/nftTutorialMinter
+
 		// store an empty NFT Collection in account storage
-        self.account.save(<-self.createEmptyCollection(), to: /storage/NFTCollection)
+        self.account.save(<-self.createEmptyCollection(), to: self.CollectionStoragePath)
```

Comment:
I think it is confusing that this is in the contract code. For all users except the user that deploys the contract this has to be done in a transaction. So why not learn people that in this tutorial?

Reply:
We do teach that in this tutorial in a transaction that we use later for account 0x01. It is just convenient to have it here also